app/views/simplified-account/settings/team-members/index.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "../../../layout.njk" %}
+
+{% macro teamMemberEmail(email, isCurrent, isServiceAdmin) %}
+  <p class="govuk-body">{{email}}{% if isCurrent %} (you){% endif %}</p>
+{% endmacro %}
+
+{% block pageTitle %}
+  Settings - Team members
+{% endblock %}
+
+{% block sideNavigation %}
+  {% include "../_settings-navigation.njk" %}
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-three-quarters">
+    <h1 class="govuk-heading-l">Team members</h1>
+    {% if isServiceAdmin %}
+      <p class="govuk-body govuk-!-margin-bottom-6 admin-lede">Team member changes affect both live and test environments.</p>
+    {% else %}
+      <p class="govuk-body govuk-!-margin-bottom-6 admin-lede">Contact an administrator to invite team members and change permissions.</p>
+    {% endif %}
+
+    {% if isServiceAdmin %}
+      {{
+      govukButton({
+        text: "Invite a team member",
+        href: inviteTeamMemberLink,
+        attributes: {
+          id: "invite-team-member-link"
+        }
+      })
+      }}
+    {% endif %}
+  </div>
+
+  <div class="govuk-grid-column-three-quarters admin-list-group govuk-!-margin-top-6">
+  {% for role, roleDetails in roles %}
+    {% if role in teamMembers %}
+      {% if teamMembers[role].length %}
+        <div class="team-member-email-list" id="team-members-{{ role }}-list">
+          <h3 id="{{ role }}-role-header" class="govuk-heading-m">
+            {{ roleDetails.description }} <span class="govuk-!-font-weight-regular pay-text-grey">({{ teamMembers[role].length }})</span>
+          </h3>
+          {% for member in teamMembers[role] %}
+            {{ govukSummaryList({
+              rows: [
+                {
+                  key: {
+                  text: "Email address",
+                  classes: "govuk-!-display-none"
+                },
+                  value: {
+                  html: teamMemberEmail(member.email, member.isCurrent)
+                },
+                  actions: {
+                  items: [
+                    { href: member.removeLink, text: "View", classes: "govuk-link--no-visited-state" }
+                  ] if member.isCurrent else [
+                  { href: member.changePermissionLink, text: "Change permission", classes: "govuk-link--no-visited-state" },
+                  { href: member.removeLink, text: "Remove", classes: "govuk-link--no-visited-state" }
+              ]
+            }
+              }
+              ]
+              }) }}
+          {% endfor %}
+        </div>
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+
+
+  {% if numberInvitedMembers > 0 %}
```

Comment:
this block is very similar to the preceding block, perhaps we could refactor the controller to including all team members (invited or active) in one collection

Reply:
Quite possibly. The structure of invited users is a bit different but I agree it could be done. My preference would be to get this PR merged before attempting further refactoring.